app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -281,5 +279,11 @@ export class ContractsController {
   _currentDataSource(){
     return reduced || data
   }
+
+  _formalizedContracts(){
+    return this._currentDataSource().contractsData.filter((contract) =>
+      contract.status == 'Formalizado' || contract.status == 'Adjudicado'
```

Comment:
Destructuring better and triple equals in comparisons `status === 'Formalizado'`

http://longgoldenears.blogspot.com/2007/09/triple-equals-in-javascript.html

Reply:
Ops, I usually puts triple equals, but old habits die hard :) thanks